e2e/conftest.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2023 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import warnings
+from datetime import datetime
+from pathlib import Path
+
+from pytest import fixture
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+
+def _get_xfail():
+    """
+    Get the XFAILS environment variable. We use a default of 1, indicating we expect
+    to add xfail marker to `test_parse_access_ncfile[AccessOm2Builder-access-om2/output000/ocean/ocean_grid.nc-expected0-True]`
+    unless specified.
```

Comment:
Probably should be updated to reference which test a value of `1` will point to.

Reply:
Good spot